.env.example

```diff
@@ -22,7 +22,10 @@ ASK_COOLDOWN_ROLE=
 
 CHANNEL_NAMES=list,help,channel,names,here,seperated,by,commas
 
-# In seconds
+# In milliseconds
```

Comment:
Replace with something like "Following variables are in milliseconds" and remove the other comments?